app/src/main.cpp

```diff
@@ -463,27 +438,12 @@ namespace scitt
               historical_state->transaction_id.to_str()));
         }
 
-        SCITT_DEBUG("Get saved SCITT entry");
```

Comment:
Leave the debug statement please

Reply:
I don't understand, there's no getting here anymore, why we would log that there is?

Reply:
I do not know which statement you please, please feel free to use the [GitHub suggestion feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) to propose a concrete change.